tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -43,7 +43,12 @@ def start_replica_cmd(builddir, replica_id):
             "-v", viewChangeTimeoutMilli,
             "-e", str(True)
             ]
+# uncomment for live tracking of log messages from the test
```

Comment:
I would suggest for us to keep this code block commented and leave the logging output to be handled by eliot by default. I think un-commenting it is only useful for live tracking of the tests, which we do on our machines.

Reply:
Done